content/katas/fibonacci-sequence/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Fibonacci sequence
+
+weight: 5
+---
+
+ **What is fibonacci sequence?**
+
+>Fibonacci Sequence: is the series of numbers where the next number is found by adding up the 2 numbers before it.
+
+**TASK**
+
+**1.** 
+Write a function, `fibonacci(n)` that takes in a number/int argument (n) and returns the first nth terms of the sequence. 
+
+```js
+function fibonacci(n)
+```
```

Comment:
Since you don't want to delete this.. you can at least make it make sense to keep it. maybe say something, to me, this is just weirdly placed. 
```js
function fibonacci(n){
     // Smart things
}
```